### DISCUSSION TITLE
Failed to load my svelteKit application

### DISCUSSION THREAD

**Discussion by Graciasjr** [sveltejs/kit#11071] [General]
I have a small problem in my svelteKit application. Recently my svelteKit application has been slow in loading layouts or when my page loads, it is never resolved. I am forced to end my application's running session before turning it back on. This problem occurs every time I save a function in a module file, in this case utils.svelte of my svelteKit application. I don't know what causes this problem Is it the amount of line of code in the module or the version of svelteKit (v.1.0.0)

**Comment by teemingc** [comment]
You'll need to provide a minimal reproduction for this issue to be troubleshooted.
